embedding.c

```diff
@@ -295,6 +295,7 @@ hnsw_gettuple(IndexScanDesc scan, ScanDirection dir)
 		int         n_items;
 		size_t      n_results;
 		label_t*    results;
+		bool        res;
```

Comment:
I think a better name would be something like `search_succeeded`

Reply:
Replaced